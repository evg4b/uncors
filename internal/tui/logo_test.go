package tui_test

import (
	"testing"

	"github.com/evg4b/uncors/testing/testutils"
	"github.com/gkampitakis/go-snaps/snaps"

	"github.com/evg4b/uncors/internal/tui"
)

func TestLogo(t *testing.T) {
	t.Run("Logo", testutils.WithTrueColor(func(t *testing.T) {
		version := "0.1.0"
		logo := tui.Logo(version)

		snaps.MatchSnapshot(t, logo)
	}))
}
