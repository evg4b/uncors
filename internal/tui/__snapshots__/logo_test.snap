
[TestLogo/Logo - 1]
[38;2;220;1;0mâ–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆâ–ˆ    â–ˆâ–ˆ [0m[38;2;255;211;0m â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ[0m
[38;2;220;1;0mâ–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆâ–ˆâ–ˆ   â–ˆâ–ˆ [0m[38;2;255;211;0mâ–ˆâ–ˆ      â–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆ   â–ˆâ–ˆ â–ˆâ–ˆ     [0m
[38;2;220;1;0mâ–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆ â–ˆâ–ˆ  â–ˆâ–ˆ [0m[38;2;255;211;0mâ–ˆâ–ˆ      â–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ[0m
[38;2;220;1;0mâ–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆ  â–ˆâ–ˆ â–ˆâ–ˆ [0m[38;2;255;211;0mâ–ˆâ–ˆ      â–ˆâ–ˆ    â–ˆâ–ˆ â–ˆâ–ˆ   â–ˆâ–ˆ      â–ˆâ–ˆ[0m
[38;2;220;1;0m â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆ   â–ˆâ–ˆâ–ˆâ–ˆ [0m[38;2;255;211;0m â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ  â–ˆâ–ˆ   â–ˆâ–ˆ â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ[0m
                                     version: 0.1.0
---
